zookeeper-server/src/test/java/org/apache/zookeeper/server/auth/znode/groupacl/X509ZNodeGroupAclProviderTest.java

```diff
@@ -115,7 +114,8 @@ public void setUp() throws Exception {
 
   @After
   public void cleanUp() throws InterruptedException, KeeperException {
-    ZNodeGroupAclProperties.clearProperties();
+    LOG.info("Tearing down...");
```

Comment:
Nit: could we say something like "X509ZNodeGroupAclProviderTest::cleanUp() called!" so it's easier to figure out what's exactly being called in the logs?